lib/ansible/modules/database/postgresql/postgresql_query.py

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Felix Archambault
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'supported_by': 'community',
+    'status': ['preview']
+}
+
+DOCUMENTATION = '''
+---
+module: postgresql_query
+short_description: Run arbitrary PostgreSQL queries.
+description:
+   - Run arbitrary PostgreSQL queries.
+   - Can read queries from a .sql script files.
+   - SQL scripts may be templated with ansible variables at execution time.
+   - >
+    Query results may be assigned to a variable for later reuse during ansible
+    runtime.
+version_added: "2.4"
+options:
+    db:
+        description:
+            - name of the database to run queries against.
+        required: false
+        default: None
+    query:
+        description:
+            - SQL query to run. Variables can be escaped with psycopg2 syntax.
```

Comment:
A link to pyscopg2 documentation could be added (using `U()` formatting function).

Reply:
I added a link. Can be handy indeed.